Frameworks-integrations/MIGraphX/Quantizations.rst

```diff
@@ -0,0 +1,145 @@
+Running Quantized ResNet50 via MIGraphX
+========================================
+
+This example walks through the dynamo PTQ workflow for running a quantized model using torch_migraphx.
```

Comment:
What is a PTQ?

Reply:
Post Training Quantization. Corrected. @saadrahim